src/hooks/use-fetch-wpcom-sites/index.tsx

```diff
@@ -1,40 +1,47 @@
 import * as Sentry from '@sentry/electron/renderer';
 import { useState, useEffect, useRef, useMemo, useCallback } from 'react';
+import { z } from 'zod';
 import { useAuth } from 'src/hooks/use-auth';
 import { reconcileConnectedSites } from 'src/hooks/use-fetch-wpcom-sites/reconcile-connected-sites';
 import { useOffline } from 'src/hooks/use-offline';
 import { getIpcApi } from 'src/lib/get-ipc-api';
 import type { SyncSite, SyncSupport } from 'src/hooks/use-fetch-wpcom-sites/types';
 
-type SitesEndpointSite = {
-	ID: number;
-	is_wpcom_atomic: boolean;
-	is_wpcom_staging_site: boolean;
-	name: string;
-	URL: string;
-	jetpack?: boolean;
-	options?: {
-		created_at: string;
-		wpcom_staging_blog_ids: number[];
-	};
-	plan?: {
-		expired: boolean;
-		features: {
-			active: string[];
-			available: Record< string, string[] >;
-		};
-		is_free: boolean;
-		product_id: number;
-		product_name_short: string;
-		product_slug: string;
-		user_is_owner: boolean;
-	};
-	is_deleted: boolean;
-};
-
-type SitesEndpointResponse = {
-	sites: SitesEndpointSite[];
-};
+export const sitesEndpointSiteSchema = z.object( {
+	ID: z.number(),
+	is_wpcom_atomic: z.boolean(),
+	is_wpcom_staging_site: z.boolean(),
+	name: z.string(),
+	URL: z.string(),
+	jetpack: z.boolean().optional(),
+	is_deleted: z.boolean(),
+	options: z.object( {
+		created_at: z.string(),
+		wpcom_staging_blog_ids: z.array( z.number() ),
+	} ),
```

Comment:
Hm, is it intentional to be not optional now?
```suggestion
	} ).optional(),
```

Reply:
It is indeed 👍 I looked closer at the API, and `options` should always be defined for us